docker-compose.yml

```diff
@@ -77,9 +77,12 @@ services:
     build:
```

Comment:
note: i made PR #279 because the seed everything PR did somehow revert changes that should have not been revert. I'll check this file again after you merged main into this branch